x-pack/plugins/observability/docs/openapi/slo/components/schemas/slo_response.yaml

```diff
@@ -14,6 +14,13 @@ properties:
     type: string
     example: My SLO description
   indicator:
+    discriminator:
+      propertyName: type
+      mapping:
+        sli.apm.transactionErrorRate: './indicator_properties_apm_availability.yaml'
+        sli.kql.custom: './indicator_properties_custom_kql.yaml'
+        sli.apm.transactionDuration: './indicator_properties_apm_latency.yaml'
+        sli.apm.sli.metric.custom: './indicator_properties_custom_metric.yaml'
```

Comment:
👀 [Interesting](https://redocly.com/docs/resources/discriminator/), I was not aware of discriminator. Nice find.